.github/workflows/build-and-push-image-base-spack.yml

```diff
@@ -8,6 +8,13 @@ on:
         type: string
         default: "main"
         description: "Either a git tag or branch name for the ACCESS-NRI/spack_packages repository, which defaults to the main branch"
+  workflow_call:
```

Comment:
Just out of interest, whats the difference between workflow_call vs workflow_dispatch?

Reply:
Workflow_dispatch means that this workflow can be called via the GitHub CLI, the web interface, or the GitHub API
Workflow_call means that this workflow can be called via other workflows (or it can be turned into an action and used with the `uses: actions/some_action` keyword.

Reply:
Ok, thank you for that explanation!